### ISSUE TITLE
FFx, and other browse that backtrace, indicates the error originated from the adapter rather then the actual code

### ISSUE THREAD

**Issue by ibolmo** [ibolmo/jasmine-jstd-adapter#3]

**Comment by reesd** [comment]
I see you reported this yourself a year ago, so can I assume you are not going to fix it anytime soon?
It's a pretty serious limitation (for me at least), and unfortunately I think its going to force me to unfortunately use something else other than jasmine (its a new project so I can choose).
I looked at the adapter code a bit, but I admit its not obvious to me what to do either.
Thanks, d

**Comment by ibolmo** [comment]
It can be annoying, I must say. For now, I've been writing tests and running jstd immediately so I have an idea of where the errors are coming from. 

If I had time, I'd look into https://github.com/lsmith/javascript-stack-trace/blob/master/getStackTrace.js and see if that'll be useful. Perhaps capture a trace myself and throw it, or log it, so that we have a better trace. 

I haven't decided if it was JS issue or a JSTD issue (since the trace is very very simple).

**Comment by reesd** [comment]
Good news, after spending way too much time looking through the code I realized the real problem was that Jasmine was swallowing the JSTD fail message you were raising. I updated the adapter to take that out, and now on FF and Chrome the actually test's file/line is the 4th item listed in the stack. IE never showed a stack so there is nothing we can do there.
d
PS, I have some thoughts on how to make it the first item on the stack for FF and Chrome, but I need to get back to my real work. Might look at it again later.
